Only in AFLplusplus/include: afl-as.h
diff --color -ru AFLplusplus/include/afl-fuzz.h afl-mm-depend/include/afl-fuzz.h
--- AFLplusplus/include/afl-fuzz.h	2024-11-02 13:27:38
+++ afl-mm-depend/include/afl-fuzz.h	2024-11-02 13:50:24
@@ -855,6 +855,8 @@
   u32   bitsmap_size;
 #endif
 
+  bool record_sampling;
+
 } afl_state_t;
 
 struct custom_mutator {
@@ -1096,6 +1098,11 @@
    * @param data pointer returned in afl_custom_init by this custom mutator
    */
   void (*afl_custom_post_run)(void *data);
+
+  /**
+   * An additional function for end job of custom mutator
+   */
+  void (*afl_custom_end_job) (void *data);
 
   /**
    * Allow for additional analysis (e.g. calling a different tool that does a
Only in AFLplusplus/include: afl-mutations.h
Only in AFLplusplus/include: afl-persistent-replay.h
Only in AFLplusplus/include: afl-prealloc.h
Only in AFLplusplus/include: afl-record-compat.h
Only in AFLplusplus/include: alloc-inl.h
Only in AFLplusplus/include: android-ashmem.h
Only in AFLplusplus/include: cmplog.h
Only in AFLplusplus/include: common.h
diff --color -ru AFLplusplus/include/config.h afl-mm-depend/include/config.h
--- AFLplusplus/include/config.h	2024-11-02 13:27:38
+++ afl-mm-depend/include/config.h	2024-11-02 13:27:20
@@ -47,7 +47,7 @@
    switches to exploitation mode. It automatically switches back when new
    coverage is found.
    Default: 300 (seconds) */
-#define STRATEGY_SWITCH_TIME 1000
+#define STRATEGY_SWITCH_TIME 0
 
 /* Default file permission umode when creating files (default: 0600) */
 #define DEFAULT_PERMISSION 0600
@@ -511,7 +511,7 @@
    but do not use them as seeds for fuzzing. This is useful for conveniently
    measuring coverage that could be attained by a "dumb" fuzzing algorithm: */
 
-// #define IGNORE_FINDS
+#define IGNORE_FINDS
 
 /* Text mutations */
 
Only in AFLplusplus/include: coverage-32.h
Only in AFLplusplus/include: coverage-64.h
Only in AFLplusplus/include: debug.h
Only in AFLplusplus/include: envs.h
Only in AFLplusplus/include: forkserver.h
Only in AFLplusplus/include: hash.h
Only in AFLplusplus/include: list.h
Only in AFLplusplus/include: sharedmem.h
Only in AFLplusplus/include: snapshot-inl.h
Only in AFLplusplus/include: t1ha.h
Only in AFLplusplus/include: t1ha0_ia32aes_b.h
Only in AFLplusplus/include: t1ha_bits.h
Only in AFLplusplus/include: t1ha_selfcheck.h
Only in AFLplusplus/include: types.h
Only in AFLplusplus/include: xxhash.h
